packages/quick-edit/package.json

```diff
@@ -25,6 +25,6 @@
 		"wrangler": "workspace:*"
 	},
 	"engines": {
-		"node": "<17"
+		"node": ">=18.15"
```

Comment:
not `18.18`? What does this version number depend on?

Reply:
Thanks for catching this! This should probably be 18.18 too